src/MSBuildLocator/MSBuildLocator.cs

```diff
@@ -8,6 +8,7 @@
 using System.Linq;
 using System.Reflection;
 using System.Text;
+using System.ComponentModel;
```

Comment:
Looking at the existing list, are these normally sorted alphabetically?

Reply:
Fixed, thanks